app/presenters/hyrax/file_set_presenter.rb

```diff
@@ -96,7 +96,7 @@ def link_presenter_class
       end
 
       def fetch_parent_presenter
-        ids = ActiveFedora::SolrService.query("{!field f=member_ids_ssim}#{id}",
+        ids = Hyrax::SolrService.query("{!field f=member_ids_ssim}#{id}",
                                               fl: ActiveFedora.id_field)
```

Comment:
Rubocop is complaining about the alignment for lines such as these (please see https://circleci.com/gh/samvera/hyrax/4213?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).